README.md

```diff
@@ -58,6 +58,44 @@ console.log(config)
 **NB:** internally this plugin force to not have additional properties,
 so the `additionalProperties` flag is forced to be `false`
 
+### Custom keywords
+This library supports the following Ajv custom keywords:
+
+#### `seperator`
+Type: `string`
+
+Applies to type(s): `string`
```

Comment:
```suggestion
Applies to type: `string`
```

Reply:
👍 

I also realized there is a typo in the docs for the keyword name. It should be `separator`.